src/Microsoft.IdentityModel.Tokens/ECDsaAdapter.cs

```diff
@@ -244,7 +238,7 @@ private static bool SupportsCNGKey()
 #pragma warning restore CA1416 // Validate platform compatibility
                 return true;
             }
-            catch
+            catch (PlatformNotSupportedException)
```

Comment:
also an error in the IDE to not have a specified exception to catch. Going with `PlatformSupportedException` as we are suppressing a platform compatibility warning and using `CngKeyBlobFormat.EccPrivateBlob` to determine if CNG (Cryptography Next Generation) keys are supported. But not sure if this is the right exception type here. 🤷

Reply:
This method is not used anymore, it was used with .net451.

Reply:
removed